vue-js/src/main.ts

```diff
@@ -0,0 +1,18 @@
+import {createApp} from 'vue'
+import { createPinia } from 'pinia'
+
+import App from './App.vue'
+import router from './router'
+import ScanbotSDK from "scanbot-web-sdk";
+
+import './assets/main.css'
+
+const app = createApp(App)
+
+const scanbotSDK = ScanbotSDK.initialize({ licenseKey: "", engine: "/scanbot-web-sdk/" });
```

Comment:
Is that engine path valid after a fresh clone? Is there a problem copying wasm files with the default path?